yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1196,23 +1197,23 @@ func fetchUnsupportedPlPgSQLObjects(schemaAnalysisReport utils.SchemaReport) []U
 	plpgsqlIssues := lo.Filter(schemaAnalysisReport.Issues, func(issue utils.AnalyzeSchemaIssue, _ int) bool {
 		return issue.IssueType == UNSUPPORTED_PLPGSQL_OBJECTS_CATEGORY
 	})
-	groupPlpgsqlIssuesByReason := lo.GroupBy(plpgsqlIssues, func(issue utils.AnalyzeSchemaIssue) string {
-		return issue.Reason
+	groupPlpgsqlIssuesByIssueName := lo.GroupBy(plpgsqlIssues, func(issue utils.AnalyzeSchemaIssue) string {
+		return issue.Name
```

Comment:
Whenever applying an comparison logic, it would be better to use `Type` rather than `Name`/`Description`/etc

Reply:
The issue with grouping by reason(/description) anymore is that it is dynamic(`%s`s), having column info etc..

There will be new unnecessary groups if we go that route for different value of %s in each case.

Reply:
Right, but `type` which would be static. So, grouping by `type` should work, no?

Reply:
@makalaaneesh  I tried doing that, but here problem is:
1. Grouping for unsupported features, needs Name not Type
2. We can group by Type, save the Name in a var and put that in the unsupported feature, but that doesn't seem natural to me.

I can add a TODO if that sound okay, to get rid of the unflattened struct members.